src/App.js

```diff
@@ -14,11 +17,77 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+export default class App extends React.Component {
+  state = {
+    countGoods: 0,
+    nameGood: '',
+    arrSelectedItem: [],
```

Comment:
```suggestion
    countGoods: 0,
    selectedGoods: [],
```
Храни все туду в массиве, и через .join выводи на страничку